keps/sig-node/1769-memory-manager/README.md

```diff
@@ -138,6 +134,15 @@ In the admission phase, the *Memory Manager* uses `Allocate()` and updates its `
   - using hugepages as memory backing mechanism to avoid double-memory-management via the guest and the host kernel
   - I/O devices passthrough, where it makes sense to run on NUMA nodes nearby the device
 
+### Notes/Constraints/Caveats (Optional)
```

Comment:
Maybe we can add that the state file needs to be removed in case memory manager policy is updated and kubelet is restarted.

Another point we can mention here is that the scheduler is topology unaware and therefore a pod can fail at admission time in case resource alignment is not possible .

Also, the Risks and Mitigation section below is not populated.

Reply:
The state file is mentioned a couple times below, I think is sufficient (but I'm open to comments here). Adding a note about admission is a good idea, I'll do that.